test/validator/samples/a11y-autocomplete-valid/input.svelte

```diff
@@ -0,0 +1,21 @@
+<!-- VALID -->
+<input type="text" />
+<input type="text" autocomplete="name" />
+<input type="text" autocomplete="off" />
+<input type="text" autocomplete="on" />
+<input type="text" autocomplete="billing family-name" />
+<input type="hidden" autocomplete="section-blue shipping street-address" />
+<input type="text" autocomplete="section-somewhere shipping work email" />
+<input type="text" autocomplete="section-somewhere shipping work email webauthn" />
+<input type="text" autocomplete="SECTION-SOMEWHERE SHIPPING WORK EMAIL WEBAUTHN" />
+<input type="TEXT" autocomplete="ON" />
+<input type="email" autocomplete="url" />
+<input type="text" autocomplete="section-blue shipping street-address" />
+
+<!-- INVALID -->
+<input type="text" autocomplete />
```

Comment:
`axe-core` does not have this as a test case.

https://github.com/dequelabs/axe-core/blob/c7957d2e8be4bfd5c9aa678036fb945dd23a9c27/test/integration/rules/autocomplete-valid/autocomplete-valid.html